scss/overrides/_type.scss

```diff
@@ -0,0 +1,18 @@
+a {
+  text-decoration: none;
+}
+
+.learn-more {
+  color: $dark-blue !important;
```

Comment:
please don't use `!important`

Reply:
For this, it's a one off and is inheriting the color of the h2 element. What would be a better way to target this specific element?

Reply:
btw this class is for the Pulse tagline